inspire/dojson/hep/schemas/hep-0.0.1.json

```diff
@@ -593,6 +612,45 @@
                 }
             }
         },
+        "succeeding_entry": {
+            "type": "object",
+            "properties": {
+                "relationship_code": {
+                    "type": "string"
+                },
+                "recid": {
+                    "type": "string"
+                },
+                "isbn": {
+                    "type": "string"
+                }
+            }
+        },
+        "acquisition_source": {
+            "type": "object",
+            "properties": {
+                "source": {
+                    "type": "string",
+                    "description": "Specific volume number"
+                },
+                "email": {
+                    "type": "string",
+                    "description": "Title of the book series"
```

Comment:
I assume this is referring to book series, now in 490? Then it should be called book_series instead of acquisition_Source. And then maybe just "volume" or "volume number" and "series title"

- Annette

On Oct 7, 2015, at 10:17 AM, Javier Martin Montull <notifications@github.com<mailto:notifications@github.com>> wrote:


In inspire/dojson/hep/schemas/hep-0.0.1.json<https://github.com/inspirehep/inspire-next/pull/400#discussion_r41362721>:

> +                },
> +                "isbn": {
> +                    "type": "string"
> +                }
> +            }
> +        },
> +        "acquisition_source": {
> +            "type": "object",
> +            "properties": {
> +                "source": {
> +                    "type": "string",
> +                    "description": "Specific volume number"
> +                },
> +                "email": {
> +                    "type": "string",
> +                    "description": "Title of the book series"


The descriptions seem to be off here.

—
Reply to this email directly or view it on GitHub<https://github.com/inspirehep/inspire-next/pull/400/files#r41362721>.